### ISSUE TITLE
Recommended spawn-sync version does not exist

### ISSUE THREAD

**Issue by paulmenzel** [ForbesLindesay/sync-request#49]
Using Node.js 0.10.29, *sync-request* aborts with the error below.

```
     Error: Sync-request requires node version 0.12 or later.  If you need to use it with an older version of node
you can `npm install spawn-sync@2.2.0`, which was the last version to support older versions of node.
      at doRequest (node_modules/sync-request/index.js:15:11)
      at Object.checkBarcode (index.js:30:15)
      at Context.<anonymous> (test/index.js:6:18)
```

Trying to install [*spawn-sync*](https://github.com/ForbesLindesay/spawn-sync) 2.2.0 does fail though. According to the README the latest version is 1.0.15.

**Comment by ForbesLindesay** [comment]
It should say `syncRequest@2.2.0`

**Comment by paulmenzel** [comment]
Thank you. I’ll send a pull request.

**Comment by paulmenzel** [comment]
Ok, it’s actually already fixed with pull request #46.

Would that warrant a 3.0.1 release?
